README.md

```diff
@@ -5,17 +5,69 @@ This is a plugin to extend the [Orion] text editor to support features allowing
 ## Current Features
 
 - Context-free highlighting of Umple and extra code
+- Generation of Umple code directly in the Orion editor (Partial - see below)
 
 ## Work-in-Progress Features
 
 - Context-sensitive highlight
-- Generation of Umple code directly in the Orion editor
 - Live generation of UML diagrams of Umple code
 - Links to edit code in Orion (a la JSFiddle)
 
-## Setup
+## User Plugin Setup
 
-To set up this plugin, you must first download and host `plugin.html` and `orion.umple.js` onto a web server that an instance of Orion can pull it from. In Orion itself (either local or [hosted][Orion]), go to {Settings > Plug-ins} and click the 'Install' button on the top right. Enter the URL for `plugin.html` (with `orion.umple.js` in the same directory) and it should install without any issues.
+To set up this plugin, you must first download and host `plugin.html` and `orion.umple.js` onto a web server that an instance of Orion can retrieve it from. In Orion itself (either local or [hosted][Orion]), go to {Settings > Plug-ins} and click the 'Install' button on the top right. Enter the URL for `plugin.html` (with `orion.umple.js` in the same directory) and it should install without any issues.
+
+Do note that this plugin will not allow compilation of code using any arbitrary instance of Orion. In order to compile code in Orion, it must be prebuilt with Umple compiler support.
+
+## Server Plugin Setup
+
+For server admins who would like Umple compilation built into their Orion server instance, the following instructions will add that functionality.
+
+### For those who want to add Umple to an unmodified server
+
+Clone the orion client:
+
+    git clone https://git.eclipse.org/r/orion/org.eclipse.orion.client.git
+
+Copy the Umple.Orion files into the cloned repo
+
+    cp -r umple.orion/org.eclipse.orion.client/* org.eclipse.orion.client/
+
+Clone and build the server repo (note: the Orion Client and Server repositories must be in the same directory for Orion to build.)
+
+    git clone https://git.eclipse.org/r/orion/org.eclipse.orion.server.git
+    mvn clean install -P platform-kepler,local-build -DskipTests
```

Comment:
I tried to do build, but it looks like there must be a pom.xml file.
![screenshot from 2016-03-31 15 01 54](https://cloud.githubusercontent.com/assets/3684908/14187617/e8e88640-f751-11e5-8a5a-4d83ac1f3bc1.png)

any idea bout the error?